sympy/physics/mechanics/kane.py

```diff
@@ -331,8 +383,8 @@ def _initialize_constraint_matrices(self, config, vel, acc):
             # mxm * mxp  =  mxp
             # B_r * A_rs = -B_s
 
-            B_s = self._k_nh[:, :self.num_ind_speeds]
-            B_r = self._k_nh[:, self.num_ind_speeds:]
+            B_s = self._k_nh[:, :self.num_independent_speeds]
+            B_r = self._k_nh[:, self.num_independent_speeds:]
```

Comment:
This goes along with the question up above about choosing a standard between abbreviating and not abbreviating. The abbreviated version makes for shorter lines of code but I like the "at a glance" clarity that the non abbreviated version brings.